DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/Reports/Index.cshtml

```diff
@@ -43,38 +43,25 @@
          role="button">
         Download usage data
       </a>
+
+      <hr />
+      <h2 class="nhsuk-heading-m">Evaluation summary</h2>
+      <p>Summary of evaluation responses collected from delegates on completion of learning activities.</p>
+
+      <ul class="nhsuk-card-group">
+        @foreach (var evaluationSummary in Model.EvaluationSummaryBreakdown) {
+          <li class="nhsuk-card-group__item">
+            <partial name="_EvaluationSummaryCard.cshtml" model="evaluationSummary" />
+          </li>
+        }
+      </ul>
     } else {
-      var place =
-        Model.ReportsFilterModel.CourseCategoryName == "All"
-          ? "at this centre"
-          : $"for the course category {Model.ReportsFilterModel.CourseCategoryName}";
+      var place = Model.Category == "All" ? "at this centre" : $"for course category {Model.Category}";
       <p class="nhsuk-body-l">
         There has not yet been any activity @place.
       </p>
     }
 
-    <hr />
-    <h2 class="nhsuk-heading-m">Evaluation summary</h2>
-    <p>Summary of evaluation responses collected from delegates on completion of learning activities.</p>
-
-    <ul class="nhsuk-card-group">
-      @foreach (var evaluationSummary in Model.EvaluationSummaryBreakdown) {
-        <li class="nhsuk-card-group__item">
-          <partial name="_EvaluationSummaryCard.cshtml" model="evaluationSummary" />
-        </li>
-      }
-    </ul>
-
-    @if (Model.HasActivity) {
-      <a class="nhsuk-button"
-         asp-controller="Reports"
-         asp-action="DownloadEvaluationSummaries"
-         asp-all-route-data="@Model.ReportsFilterModel.FilterValues"
-         role="button">
-        Download evaluation summary
-      </a>
-    }

```

Comment:
Was it intentional to completely delete the "Download evaluation summary" button in all circumstances?

Reply:
No, nice catch!